src/test/java/seedu/duke/flashutils/utils/StorageTest.java

```diff
@@ -0,0 +1,48 @@
+package seedu.duke.flashutils.utils;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import seedu.duke.flashutils.types.Card;
+import seedu.duke.flashutils.types.FlashBook;
+import seedu.duke.flashutils.types.FlashCardSet;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StorageTest {
+
+    @Test
+    void writeAndReadFlashCardsTest() throws IOException {
```

Comment:
If the test case isn’t self-explanatory from the method name, a brief comment explaining the purpose/nature/expected behaviours of the test can be helpful.